diff --git a/configure b/configure
index af65be1..20c4bbf 100755
--- a/configure
+++ b/configure
@@ -14650,17 +14650,6 @@ else
 $as_echo "no" >&6; }
 fi
 
-
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for gtester2xunit dependencies" >&5
-$as_echo_n "checking for gtester2xunit dependencies... " >&6; }
-if !($PYTHON -c "import libxslt, libxml2" 2> /dev/null); then
-  { $as_echo "$as_me:${as_lineno-$LINENO}: result: no" >&5
-$as_echo "no" >&6; }
-  as_fn_error $? "You need to install python-libxslt1 and python-libxml2" "$LINENO" 5;
-fi
-{ $as_echo "$as_me:${as_lineno-$LINENO}: result: yes" >&5
-$as_echo "yes" >&6; }
-
 ###########################
 # Headless tests
 ###########################
